bedevere/bpo.py

```diff
@@ -40,16 +33,22 @@
     issue_number_found = ISSUE_RE.search(event.data["pull_request"]["title"])
     if not issue_number_found:
         issue = await util.issue_for_PR(gh, event.data["pull_request"])
-        status = SKIP_ISSUE_STATUS if util.skip("issue", issue) else FAILURE_STATUS
+        status = (SKIP_ISSUE_STATUS if util.skip("issue", issue)
+                                    else create_failure_status_no_issue())
     else:
-        if "body" in event.data["pull_request"]:
-            body = event.data["pull_request"]["body"] or ""
-            if not body or CLOSING_TAG not in body:
-                issue_number = issue_number_found.group("issue")
-                new_body = BODY.format(body=body, issue_number=issue_number)
-                body_data = {"body": new_body}
-                await gh.patch(event.data["pull_request"]["url"], data=body_data)
-        status = create_success_status(issue_number_found)
+        issue_number = issue_number_found.group("issue")
+        session = kwargs.get('session', '')
```

Comment:
Instead of doing this I would add `session` as keyword argument to this function with a default value of `None`.

Reply:
Of course!